web/src/components/pages/dashboard/stats/leaderboard-card.tsx

```diff
@@ -127,6 +127,9 @@ class UnconnectedLeaderboard extends React.Component<Props, State> {
   }
 
   playAvatarClip = function(clipUrl: string, position: any) {
+    const { locale } = this.props;
+    trackVoiceAvatar('listen', locale);
```

Comment:
a bit of an edge-casy nit: could we trigger self-listen here when a user clicks on their own row?

Reply:
yeah! :+1: